bin/node/rpc/src/lib.rs

```diff
@@ -159,6 +160,10 @@ where
 		finality_provider,
 	)));
 
+	io.extend_with(substrate_state_trie_migration_rpc::StateMigrationApi::to_delegate(
```

Comment:
should we have at least a companion that adds this temporarily to polkadot binary as well well?

Reply:
I will update my branch (latest change where not in).

Reply:
@kianenigma https://github.com/paritytech/polkadot/compare/master...cheme:state-check-rpc